04_verify.sh

```diff
@@ -0,0 +1,184 @@
+#!/usr/bin/env bash
+
+set -u
+
+exec 3>&1
+
+source lib/logging.sh
+source lib/common.sh
+
+KUBECONFIG=~/.kube/config
+EXPTD_CRDS="baremetalhosts.metal3.io clusters.cluster.k8s.io \
+  machineclasses.cluster.k8s.io machinedeployments.cluster.k8s.io \
+  machines.cluster.k8s.io machinesets.cluster.k8s.io"
+EXPTD_STATEFULSETS="cluster-api-controller-manager \
```

Comment:
These definitions can be moved after function definitions so that all global stuff remains close.